__tests__/integration/fixtures/bug-include-path-edit-uri/mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: "Example"
+site_description: "Description Here"
+repo_url: https://github.com/backstage/mkdocs-monorepo-plugin
+edit_uri: edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/
```

Comment:
Oh!  One last note, in case I forget before we merge: this was super helpful for understanding what was working/not working in this PR, but just before we merge, we should update so that it uses the master branch.

Reply:
Can I create a separate pull request after merging this one to do this?